src/components/HomeLayout/LibrariesSection.tsx

```diff
@@ -0,0 +1,209 @@
+import React, { FC } from 'react';
+import Link from '@docusaurus/Link';
+import {
+  Identity,
+  IotaCore,
+  Streams,
+  Stronghold,
+  Wallet,
+} from '../../icons';
+
+interface LanguagesProps {
+  languages?: { [key: string]: string };
+}
+
+const Languages: FC<LanguagesProps> = (props) => {
+  return (
+    <>
+      {Object.entries(props.languages).length > 0 && (
+        <div className='languages'>
+          {Object.entries(props.languages).map(([language, to], key) => (
+            <Link
+              className='language button button--outline button--primary'
+              to={to}
+              key={key}
+            >
+              {language}
+            </Link>
+          ))}
+        </div>
+      )}
+    </>
+  );
+};
+
+const LibrariesSection: FC = () => (
+  <div className='libraries__cards'>
+    <div className='libraries__card libraries__card--identity'>
+      <img
+        className='libraries__logo'
+        src='/img/homepage/identity_logo.svg'
+      />
+      <div className='libraries__section'>
+        <div className='libraries__head'>
+          <div className='libraries__icon'>
+            <Identity />
+          </div>
+          <Languages
+            languages={{
+              Rust: 'identity.rs/libraries/rust/getting_started',
+              WASM: 'identity.rs/libraries/wasm/getting_started',
+            }}
+          />
+        </div>
+        <h3 className='libraries__header'>Identity</h3>
+        <ul className='libraries__features'>
+          <li className='libraries__feature'>Identity framework</li>
+          <li className='libraries__feature'>
+            Decentralized Identifiers (DID)
+          </li>
+          <li className='libraries__feature'>
+            Verifiable Credentials (VC)
+          </li>
+        </ul>
+        <Link
+          to='identity.rs/introduction'
+          className='libraries__button button button--outline button--primary'
+        >
+          Create an identity
+        </Link>
```

Comment:
```suggestion
        <ul className='libraries__features'>
          <li className='libraries__feature'><a href={"'/identity.rs/getting_started/create_and_publish"}>Identity framework</a></li>
          <li className='libraries__feature'>
            <a href={"/identity.rs/concepts/decentralized_identifiers/overview"}>Decentralized Identifiers (DID)</a>
          </li>
          <li className='libraries__feature'>
            <a href={"/identity.rs/concepts/verifiable_credentials/overview"}>Verifiable Credentials (VC)</a>
          </li>
        </ul>
        <Link
          to='/identity.rs/getting_started/create_and_publish'
          className='libraries__button button button--outline button--primary'
        >
          Create an identity
        </Link>
```

Reply:
I left the CTAs as plain text for now, if still applicable we can open a new issue and PR.